src/openrct2/rct1/S4Importer.cpp

```diff
@@ -2499,7 +2499,7 @@ namespace RCT1
         {
             const auto originalString = _s4.string_table[(stringId - USER_STRING_START) % 1024];
             auto originalStringView = std::string_view(
-                originalString, GetRCT2StringBufferLen(originalString, USER_STRING_MAX_LENGTH));
```

Comment:
Clearly this isn't an RCT2 function but an RCT12 function to be moved in future pr.